### PR TITLE
cvsb-183 && cvsb-184

### PR DESCRIPTION
**Pull request by mgherceanu**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,23 +48,25 @@
...
+    "cordova-plugin-file": "^6.0.1",
+    "cordova-plugin-inappbrowser": "^2.0.2",
     "cordova-plugin-ionic-webview": "^2.3.1",
-    "cordova-plugin-splashscreen": "^4.0.3",
-    "cordova-plugin-whitelist": "^1.3.1",
-    "cordova-plugin-x-socialsharing": "^5.4.1",
-    "cordova-sqlite-storage": "^2.5.0",
-    "cordova-wheel-selector-plugin": "1.1.1",
-    "es6-promise-plugin": "^4.1.0",
+    "cordova-plugin-network-information": "git+https://github.com/apache/cordova-plugin-network-information.git",
```
Comment (danieliordache) [inline package.json]: remove this package

File: src/pages/testing/test-creation/add-preparer/add-preparer.ts [hunk 2]
```diff
@@ -21,25 +21,26 @@ export class AddPreparerPage implements OnInit {
...
+              private viewCtrl: ViewController) {
     this.testReport = this.testReportService.getTestReport();
   }
 
   ngOnInit() {
     this.getPreparers();
   }
 
+  ionViewWillEnter() {
+    this.viewCtrl.setBackButtonText('Vehicle details');
```
Comment (danieliordache) [inline src/pages/testing/test-creation/add-preparer/add-preparer.ts]: create an enum for 'Vehicle Details' string

File: src/pages/testing/test-creation/vehicle-lookup/vehicle-lookup.html [hunk 3]
```diff
@@ -1,18 +1,21 @@
...
-        </ion-buttons>
-        <ion-title margin-left>
-            Identify Vehicle
-        </ion-title>
-    </ion-navbar>
-    <ion-toolbar>
-        <ion-searchbar [(ngModel)]="searchVal" placeholder="Enter registration or VIN" (search)="searchVehicle($event)"></ion-searchbar>
-    </ion-toolbar>
+  <ion-navbar hideBackButton="true">
+    <ion-buttons start>
```
Comment (danieliordache) [inline src/pages/testing/test-creation/vehicle-lookup/vehicle-lookup.html]: do it as on preparers page. If it's possible

Comment (mgherceanu) [reply src/pages/testing/test-creation/vehicle-lookup/vehicle-lookup.html]: we have a custom button there in the design, with no back arrow and different color. so i sugest we keep it as it is.

File: src/pages/testing/test-creation/add-preparer/add-preparer.ts (hunk 2 above)
Comment (mgherceanu) [reply src/pages/testing/test-creation/add-preparer/add-preparer.ts]: i have created an enum for application strings. and placed all the texts in it.

File: package.json (hunk 1 above)
Comment (mgherceanu) [reply package.json]: The plugin 'cordova-plugin-network-information' is required by (phonegap-plugin-mobile-accessibility), skipping uninstallation. (try --force if trying to update)

File: src/pipes/filter-test-type-by-vehicle/filter-test-type-by-vehicle.pipe.ts [hunk 4]
```diff
@@ -19,10 +19,10 @@ export class FilterTestTypeByVehiclePipe implements PipeTransform {
   }
 
   filterByVehicle(elem: TestTypesModel, vehicle: VehicleModel): TestTypesModel | boolean {
-    if (elem.forVehicleType.indexOf(vehicle.getType()) == -1) return false;
-    if (elem.forVehicleSize.indexOf(vehicle.getSize()) == -1) return false;
-    if (elem.forVehicleConfiguration.indexOf(vehicle.getConfiguration()) == -1) return false;
-    if (elem.forVehicleAxles && elem.forVehicleAxles.indexOf(vehicle.getAxels()) == -1) return false;
+    if (elem.forVehicleType.indexOf(vehicle.techRecord[0].vehicleType.toLowerCase()) == -1) return false;
```
Comment (acozianu) [inline src/pipes/filter-test-type-by-vehicle/filter-test-type-by-vehicle.pipe.ts]: Please put vehicle.techRecord[0].vehicleType.toLowerCase() in a variable and reuse it

Comment (mgherceanu) [reply src/pipes/filter-test-type-by-vehicle/filter-test-type-by-vehicle.pipe.ts]: done for vehicle.techRecord[0]

**Comment by tealorg** [comment]
Fine by me

**Pull request merged**